docs/pages/push-notifications/push-notifications-setup.mdx

```diff
@@ -155,43 +159,31 @@ export default function App() {
 }
 ```
 
-</Step>
-
-<Step label="3">
-
-## Test using Expo Go and push notifications tool
-
-> This step allows testing the push notifications when developing your project using Expo Go. If you are using EAS or [development builds](/development/introduction), you can skip this step and move on to the next one to configure credentials for Android and iOS.
+### Configure `projectId`
```

Comment:
```suggestion
### Configure `projectId`

> This step is only valid if you are using EAS Build. If you are not using EAS Buid, you can skip this step.
```

Reply:
I beleive that with the project id manually set, notifications become partially broken in Expo Go.

I have manually set the project id in my `app.json`:

```json
    "extra": {
      "eas": {
        "projectId": "xxxxxxx"
      }
    }
```

I then set this when getting a token:

```javascript
token = await Notifications.getExpoPushTokenAsync({
  projectId: Constants.expoConfig.extra?.eas?.projectId,
});
```

At this point notifications are only working on iOS when the app is killed or in the background. If the app is in the foreground my `handleNotification` handler never gets called, so the notification doesn't show in the foreground, `useLastNotificationResponse` is always `null`, and the `addNotificationReceivedListener` and `addNotificationResponseReceivedListener` are never called.

I beleive with the setup mentioned in this new guide that notifications will _only_ work for EAS development builds. This is fine, but I think it should at least be noted in the documentation. Either that or it should be possible to set this to a value that will work for the Expo Go client to make development easier.

Reply:
>At this point notifications are only working on iOS when the app is killed or in the background. If the app is in the foreground my handleNotification handler never gets called, so the notification doesn't show in the foreground, useLastNotificationResponse is always null, and the addNotificationReceivedListener and addNotificationResponseReceivedListener are never called.

Are you using `projectId` with Expo Go and without creating a development build (EAS Build)? In that case, you don't need `projectId`.

The guide considers development build as first class (and yes, you are right. I'll update the guide to add more context about development build a requirement).

Reply:
> Are you using projectId with Expo Go and without creating a development build (EAS Build)? In that case, you don't need projectId.

Yes, that was the situation I was in and removing the project ID lead to notification working correctly in Expo Go. I was however getting the warning that not including a project id will be unsupported from Expo 49+.

I presume it just won't be possible to do notifications using Expo Go in the future and you'll need to build a development client. Is that correct?

Reply:
We consider `expo-notifications` a part of EAS services. So yes, `projectId` is required (as the warning message states) and will be a requirement from SDK 49.

Reply:
Ok great, that's good to know for the future.

Thanks for the help and the great documentation!